mobile/lib/features/home/view/home_page.dart

```diff
@@ -39,11 +39,11 @@ class _HomePageState extends State<HomePage> {
         child: Column(
           mainAxisSize: MainAxisSize.min,
           children: <Widget>[
-            Avatar(photo: user.photo),
+            Avatar(photo: user.photoURL),
```

Comment:
Does the user have a photo?

Reply:
It's firebase user and he might photo from Google, not in our case though.